feature-libs/asm/styles/components/_asm-bind-cart.component.scss

```diff
@@ -85,16 +84,72 @@
     margin-inline-start: 6px;
     margin-inline-end: 15px;
     max-width: 20rem;
+    padding-inline-end: 15px;
+
+    @media (max-width: 575px) {
+      margin-inline-end: 0;
+      margin-inline-start: 0;
+      min-width: 100%;
+      max-width: 100%;
+    }
+    &:focus,
+    &:focus-within,
+    &:active,
+    &.active {
+      background-color: #ffffff;
+      outline: none;
+    }
+    .cx-asm-reset {
+      visibility: hidden;
+      background-color: transparent;
+      border: none;
+      color: inherit;
+    }
+  }
+
+  .cx-asm-assignCart-feature-enabled {
```

Comment:
Since there will be a day `.cx-asm-assignCart` is removed and only `.cx-asm-assignCart-feature-enabled` is left.  Maybe need a more proper name for `.cx-asm-assignCart-feature-enabled `.

Reply:
How about .cx-asm-assignCart-input-show-no-button?